webrender_traits/src/types.rs

```diff
@@ -424,6 +442,7 @@ pub trait RenderNotifier: Send {
     fn new_frame_ready(&mut self);
     fn new_scroll_frame_ready(&mut self, composite_needed: bool);
     fn pipeline_size_changed(&mut self, pipeline_id: PipelineId, size: Option<LayoutSize>);
+    fn external_event(&mut self, _evt: ExternalEvent) { unimplemented!() }
```

Comment:
This is only implemented in Gecko. Adding a default implementation that panics is to make sure we don't accidentally start using it in servo and to avoid making this a breaking change for servo.